### PR TITLE
Add index.js file.

### PR DESCRIPTION
**Pull request by SgbVGvQz-2**
Can we add this new file to repository?

### CODE REVIEW

**Comment by SgbVGvQz-1** [comment]
Can you add some more description?

**Comment by SgbVGvQz-2** [comment]
Please check now.

File: index.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// This is a test comment.
```
Comment (SgbVGvQz-1) [inline index.js]: Modify this comment.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+alert('Hello World');
```
Comment (SgbVGvQz-1) [inline index.js]: This looks good.

**Pull request merged**